### PR TITLE
Fix admin user profile's detail view

### PR DESCRIPTION
**Pull request by Leusev**
### Purpose

As you're logged in as an admin, when you want to access to user's processes, you got an error.

#### How to replicate

1. Login as an admin
2. Click on "Users" menu's option on left
3. In users list, click on the right "eye" icon of whatever user

### :camera: Screenshots

image

### CODE REVIEW

File: app/decorators/decidim/admin/users_controller_decorator.rb [hunk 1]
```diff
@@ -31,26 +31,27 @@ def show
...
         end
-      else 
+      else
         type = t("models.user.fields.process_type", scope: "decidim.admin")
       end
       process_title = process.title[locale]
       if process_title == '' then
         process_title = process.title["ca"]
       end
+      byebug
```
Comment (tramuntanal) [inline app/decorators/decidim/admin/users_controller_decorator.rb:44]: to remove please

**Review by tramuntanal — CHANGES_REQUESTED** [review]
remove the byebug please :wink:

File: app/decorators/decidim/admin/users_controller_decorator.rb (hunk 1 above)
Comment (Leusev) [reply app/decorators/decidim/admin/users_controller_decorator.rb:44]: Sorry @tramuntanal , I'll fix it now

**Review by tramuntanal — APPROVED** [review]
Thanks @Leusev !

**Pull request merged**